### PR TITLE
feat(condo): DOMA-5578 create ExternalTokenAccessRight schema

### PR DESCRIPTION
**Pull request by VKislov**
The scheme is needed to regulate the service user's access to the `accessToken` stored in the `Redis`.

### CODE REVIEW

**Comment by VKislov** [comment]
This pr requires https://github.com/open-condo-software/condo/pull/2943

File: apps/condo/domains/organization/schema/TokenAccessRight.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { SERVICE_USER_FIELD } = require('@condo/domains/miniapp/schema/fields/accessRight')
+const access = require('@condo/domains/organization/access/TokenAccessRight')
+
+const { INTEGRATIONS_NAME, SBBOL_INTEGRATION_NAME } = require('../constants')
+
+
+const TokenAccessRight = new GQLListSchema('TokenAccessRight', {
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/TokenAccessRight.js:16]: Is assum that you will create `GetTokenService` with `getToken` mutation.
Is it no so abstract?

Comment (VKislov) [reply apps/condo/domains/organization/schema/TokenAccessRight.js:16]: Not certainly in that way. We have access tokens to user data in external services that are stored with reference to the user. Therefore, the service will always require a userId to find the right token. The service was planned to be called GetAccessTokenByUserIdService

File: apps/condo/domains/user/schema/ExternalTokenAccessRight.test.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            test('anonymous can\'t', async () => {
+                const client = await makeClient()
+
+                await expectToThrowAuthenticationErrorToObj(async () => {
+                    await createTestExternalTokenAccessRight(client, service.user, SBBOL_IDENTITY_TYPE)
+                })
+            })
+        })
+
+        describe('update', () => {
```
Comment (sitozzz) [inline apps/condo/domains/user/schema/ExternalTokenAccessRight.test.js:56]: also user can't update?

File: apps/condo/domains/user/schema/ExternalTokenAccessRight.test.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+describe('ExternalTokenAccessRight', () => {
+    let admin, service
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        service = await makeClientWithServiceUser()
+    })
+
+    describe('CRUD tests', () => {
+        describe('create', () => {
```
Comment (sitozzz) [inline apps/condo/domains/user/schema/ExternalTokenAccessRight.test.js:25]: I suppose you should specify that user can't create this object

File: apps/condo/domains/user/access/ExternalTokenAccessRight.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+// @ts-nocheck
```
Comment (sitozzz) [inline apps/condo/domains/user/access/ExternalTokenAccessRight.js:1]:
```suggestion
```

File: apps/condo/domains/user/access/ExternalTokenAccessRight.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * Generated by `createschema user.ExternalTokenAccessRight 'type:Text; user:Relationship:User:CASCADE;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { SERVICE } = require('../constants/common')
```
Comment (sitozzz) [inline apps/condo/domains/user/access/ExternalTokenAccessRight.js:8]: Absolute import here would be nice

**Review by sitozzz — COMMENTED** [review]
Could this task be done with customAccess provided to a special service user?

**Review by sitozzz — APPROVED** [review]

**Comment by VKislov** [comment]
> Could this task be done with customAccess provided to a special service user?

It seems that custom rights are used to regulate access to the `postgres` database, right there the table regulates access to the `redis` database. You can do it through custom ones, but this will be non-obvious behavior

**Pull request merged**